src/styles/content.scss

```diff
@@ -1,6 +1,9 @@
 $raise-graybox: #f5f5f5;
```

Comment:
General comment on this PR which I'm putting here for no particular reason: I noticed there are lots of small commits in the history. This isn't necessarily a bad thing, and it's definitely a good habit to commit often when you're developing. However, sometimes it's nice to keep the history we push / merge a bit cleaner by creating a more intentional set of commits. One workflow to accomplish that is to perform an interactive rebase to squash / consolidate commits as well as re-order as necessary. If this is something you wan to potentially look at and the `rebase` workflow is new for you, we can pair to do that once this PR is otherwise approved (one of our conventions is that devs refrain from rebasing during a review since it impacts local branches that may have been pulled).

Reply:
When Prabhdip was helping me out on an unrelated issue last week he also mentioned the commit conventions in the engineering docs. I've since made my commits align with the doc and intended on cleaning up the commit history after the PR is approved. I have only used rebase maybe a handful of times, so I think it would be very helpful to pair whenever the PR is approved so I can make sure I rebase correctly.